### PR TITLE
 Dependency Update

### PR DESCRIPTION
**Pull request by Loori-R**
Dependencies in the `package.json` file have been updated to the latest stable versions and errors encountered during the transition to new versions have been fixed.

### CODE REVIEW

File: src/components/PromQueryField.test.tsx [hunk 1]
```diff
@@ -126,29 +141,13 @@ describe('PromQueryField', () => {
...
-  it('should not run query onBlur in explore', async () => {
-    const onRunQuery = jest.fn();
-    const { container } = render(<PromQueryField {...defaultProps} app={CoreApp.Explore} onRunQuery={onRunQuery} />);
-
-    const input = getByTestId(container, 'dummy-code-input');
-    expect(input).toBeInTheDocument();
-    await userEvent.type(input, 'metric');
-    input.blur();
-    expect(onRunQuery).not.toHaveBeenCalled();
-  });
```
Comment (dmitryk-dk) [inline src/components/PromQueryField.test.tsx:141]: Do you think we should remove this test?

File: src/configuration/AzureAuthSettings.tsx [hunk 2]
```diff
@@ -15,23 +15,21 @@
...
 import { getCredentials, updateCredentials } from './AzureCredentialsConfig';
 import { AzureCredentialsForm } from './AzureCredentialsForm';
 
-export const AzureAuthSettings: FunctionComponent<HttpSettingsBaseProps> = (props: HttpSettingsBaseProps) => {
+export const AzureAuthSettings = (props: HttpSettingsBaseProps) => {
   const { dataSourceConfig, onChange } = props;
 
-  const [overrideAudienceAllowed] = useState<boolean>(
-    config.featureToggles.prometheusAzureOverrideAudience || !!dataSourceConfig.jsonData.azureEndpointResourceId
-  );
```
Comment (dmitryk-dk) [inline src/configuration/AzureAuthSettings.tsx:34]: we do not need this state?

File: src/configuration/ConfigEditor.tsx [hunk 3]
```diff
@@ -31,14 +31,14 @@ import { HelpfulLinks } from "./HelpfulLinks";
 import { LimitsSettings } from "./LimitsSettings";
 import { PromSettings } from './PromSettings';
 
-export enum DataSourceType {
-  Alertmanager = 'alertmanager',
-}
+// export enum DataSourceType {
+//   Alertmanager = 'alertmanager',
+// }
```
Comment (dmitryk-dk) [inline src/configuration/ConfigEditor.tsx:36]: i think we can remove this comment

File: src/configuration/ConfigEditor.tsx [hunk 4]
```diff
@@ -31,14 +31,14 @@ import { HelpfulLinks } from "./HelpfulLinks";
...
 
 export type Props = DataSourcePluginOptionsEditorProps<PromOptions>;
 export const ConfigEditor = (props: Props) => {
   const { options, onOptionsChange } = props;
-  const alertmanagers = Object.values(config.datasources).filter((ds) => ds.type === DataSourceType.Alertmanager);
+  // const alertmanagers = Object.values(config.datasources).filter((ds) => ds.type === DataSourceType.Alertmanager);
```
Comment (dmitryk-dk) [inline src/configuration/ConfigEditor.tsx:41]: if this is unused we can remove this comment

File: src/utils/getRefreshFromUrl.test.ts [hunk 5]
```diff
@@ -59,11 +59,11 @@ describe('getRefreshFromUrl', () => {
...
         const actual = getRefreshFromUrl({
-          urlRefresh: refresh,
+          urlRefresh: `${refresh}`,
           currentRefresh: 'currentRefresh',
-          minRefreshInterval,
-          isAllowedIntervalFn: () => isAllowedInterval,
-          refreshIntervals,
+          minRefreshInterval: `${minRefreshInterval}`,
+          isAllowedIntervalFn: () => !!isAllowedInterval,
+          refreshIntervals: Array.isArray(refreshIntervals) ? refreshIntervals.map(s => `${s}`) : undefined,
```
Comment (dmitryk-dk) [inline src/utils/getRefreshFromUrl.test.ts:66]: Is it possible do not return undefined?

**Review by dmitryk-dk — COMMENTED** [review]
Looks good! Have some comments. Please take a look at the test pipeline it failed

File: src/components/PromQueryField.test.tsx (hunk 1 above)
Comment (Loori-R) [reply src/components/PromQueryField.test.tsx]: reverted

File: src/configuration/AzureAuthSettings.tsx (hunk 2 above)
Comment (Loori-R) [reply src/configuration/AzureAuthSettings.tsx:34]: Now `disabled={dataSourceConfig.readOnly}` is used

File: src/configuration/ConfigEditor.tsx (hunk 4 above)
Comment (Loori-R) [reply src/configuration/ConfigEditor.tsx]: removed

File: src/utils/getRefreshFromUrl.test.ts (hunk 5 above)
Comment (Loori-R) [reply src/utils/getRefreshFromUrl.test.ts]: fixed

**Review by dmitryk-dk — APPROVED** [review]
LGTM! Thank you!

**Review by hagen1778 — APPROVED** [review]

**Pull request merged**